src/constants/TokenSafetyLookupTable.ts

```diff
@@ -14,40 +14,39 @@ export enum TOKEN_LIST_TYPES {
 }
 
 class TokenSafetyLookupTable {
-  dict: { [key: string]: TOKEN_LIST_TYPES } | null = null
+  dict: { [key: string]: TOKEN_LIST_TYPES } = {}
 
-  createMap() {
-    const dict: { [key: string]: TOKEN_LIST_TYPES } = {}
+  constructor() {
+    this.update()
```

Comment:
I originally received feedback that I _shouldn't_ do it this way because of pageload time. Is the perf okay here?

Reply:
I backed this out because it was causing a lot of tests to fail.
Practically, this happens at pageload anyway, because a token logo is nearly always displayed.